index.html

```diff
@@ -213,5 +213,60 @@ <h3>Skills</h3>
             </div>
             </div> 
     </section>
+    <section class="contact">
+        <img class="logo" src="./img/image_geometry_1.png" alt="Logo">
+        <div class="contact-form">
+        <h2 id="contact">Get Started</h2>
+        <p>I'm always interested in hearing
+            about new projects, so if you'd like
+             to chat please get in touch.</p>
+        <form action="https://formspree.io/f/xqkonpny" method="post">
+            <div class="name-label">
+                <p>Your name</p>
```

Comment:
- [Optional] Dear @LizanshMathur, Good adding the changes requested by the previous reviewer, I notice Your design is a bit different from a given design, As a frontend engineer we are expected to deliver a webpage identical to the given design. Please take a look into this 
 ### Your design 
![image](https://user-images.githubusercontent.com/69077061/235903486-1b841de4-d40f-492d-bda7-e59fb94079ab.png)
 
### Figma design 
![image](https://user-images.githubusercontent.com/69077061/235903991-178021e2-a5b9-4339-80f3-0256f2a3fe76.png)